src/test/jtx/impl/AMM.cpp

```diff
@@ -494,6 +494,22 @@ AMM::deposit(
     return deposit(account, jv, assets, seq, ter);
 }
 
+IOUAmount
+AMM::deposit(DepositArg const& arg)
+{
+    return deposit(
+        arg.account,
+        arg.tokens,
+        arg.asset1In,
+        arg.asset2In,
+        arg.maxEP,
+        arg.flags,
+        arg.assets,
+        arg.seq,
+        arg.tfee,
+        arg.err);
+}
+
```

Comment:
Maybe not for this PR, but it might be worth refactoring this at some point to get rid of the overloads with multiple arguments, and just use these handy structures.

Reply:
I opened a JIRA ticket for this.